### PR TITLE
fix: [PLG-2372]: Added headings for subsections and added new url for cli 

### PR DESCRIPTION
**Pull request by devinder-harness**
### Summary

* Added headings for subsections 
*  added new URL for cli 
* Added comments to commands
#### Screenshots
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/PipelineSetupStep.tsx [hunk 1]
```diff
@@ -87,54 +105,55 @@ function CLISteps({
...
+      getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step5.commands.comments.createSecret'),
+      getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step5.commands.createsecret', {
+        gitPat:
+          state.githubPat ||
+          getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step5.commands.gitpatPlaceholder')
       }),
-      getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step4.commands.creategithubcon', {
-        gitUser: state.githubUsername
+
+      getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step5.commands.comments.createGitcon'),
```
Comment (PratyushGar) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/PipelineSetupStep.tsx:128]:
```suggestion
      getString('cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.step5.commands.comments.createGitIcon'),
```

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/PipelineSetupStep.tsx [hunk 2]
```diff
@@ -22,39 +22,51 @@ export default function PipelineSetupStep({
...
             <Label>
-              {getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step3.githubusername')}
+              {getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step4.githubusername')}
             </Label>
             <TextInput
               id="githubusername"
               name="githubusername"
               defaultValue={state.githubUsername}
-              placeholder={getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step3.githubusername')}
+              placeholder={getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step4.githubusername')}
```
Comment (PratyushGar) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/PipelineSetupStep.tsx:44]:
```suggestion
              placeholder={getString('cd.getStartedWithCD.flowbyquestions.deploymentSteps.steps.step4.githubusername')}
```

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/CLISetupStep.tsx [hunk 3]
```diff
@@ -48,35 +69,26 @@ function InstallCLIInfo(): JSX.Element {
...
+                getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step2.mvharness'),
+                getString('cd.getStartedWithCD.flowbyquestions.deplopymentSteps.steps.step2.chmod')
+              ])}
+              ignoreWhiteSpaces={false}
               downloadFileProps={{ downloadFileName: 'testname', downloadFileExtension: 'xdf' }}
               copyButtonText={getString('common.copy')}
             />
           }
         />
+        <Tab id="linux" title="Linux" panel={<CLIDownloadLinux />} />
```
Comment (PratyushGar) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/CLISetupStep.tsx:91]:
```suggestion
        <Tab id="linux" title={getString('delegate.cardData.linux.name')} panel={<CLIDownloadLinux />} />
```

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/CDOnboardingWizardWithCLI.module.scss [hunk 4]
```diff
@@ -109,3 +109,11 @@
     display: flex;
   }
 }
+
+.tabsline {
```
Comment (PratyushGar) [inline src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/CDOnboardingWizardWithCLI.module.scss:113]: nit:
```suggestion
.tabsLine {
```

**Review by Sagar-Rajput7 — APPROVED** [review]

**Pull request merged**